src/bayesian/getter.jl

```diff
@@ -8,4 +8,40 @@ function predict(container::BayesianRegression{:LinearRegression}, newdata::Data
     end
     predictions = params.α' .+ X * W'
     return vec(mean(predictions, dims=2))
+end
+
+function predict(container::BayesianRegression{:LogisticRegression}, newdata::DataFrame, prediction_chain_start::Int64 = 200)
+    X = modelmatrix(container.formula, newdata)
+
+    params = get_params(container.chain[prediction_chain_start:end,:,:])
+    W = params.β
+    if isa(W, Tuple)
+        W = reduce(hcat, W)
+    end
+    z = X * W'
+    return vec(mean(container.link.link.(z), dims=2))
```

Comment:
Will it be possible to change the name of the link property? `contained.link.link` makes it a bit ambiguous.